ksqldb-functional-tests/src/test/resources/historical_plans/protobuf_-_protobuf_primitives_with_null/7.4.0_1668095424617/plan.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
Why is there a new historic plan for this test? This PR does not affect the PROTOBUF format, does it? Same question for the other two new historic plans below.

Reply:
hmm you are right. Not sure why they were created